### DISCUSSION TITLE
https://thanhle.blog/blog/state-trong-frontend-la-gi-tai-sao-nen-gioi-cai-nay

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#24] [Ideas]
# https://thanhle.blog/blog/state-trong-frontend-la-gi-tai-sao-nen-gioi-cai-nay

https://thanhle.blog/blog/state-trong-frontend-la-gi-tai-sao-nen-gioi-cai-nay

**Comment by nguyenhoaikhang37** [comment]
lần đầu đọc blog theo dạng cốt truyện như này, vừa dễ hình dung vấn đề vừa nắm đc kĩ thuật, mong anh Thành làm thêm nhiều bài viết về reactjs hơn nữa

**Reply by thanhlmm** [reply]
Thanks em. Có vẻ mọi người thích đọc về React nhiều, mà content về React anh thấy trên mạng cũng khá nhiều rồi nên muốn viết về những thứ mới hay cách nghĩ khác hơn nên cũng hơi... bí :)) nên hầu hết anh đều viết về kiến thức nền, còn ví dụ thì là React

**Reply by nguyenhoaikhang37** [reply]
Em cũng hay tìm các bài blog đọc lắm, nhưng ở mỗi người viết lại truyền tải kiến thức mỗi cách khác nhau. Riêng em thì thích nhất blog của anh, nd đọc hài hước có, kĩ thuật cũng có. Nên anh cứ mạnh tay viết nhé, em sẽ mạnh mắt để đọc kkk
